modes/longitudinal/src/index.js

```diff
@@ -123,15 +123,15 @@ function modeFactory({ modeConfiguration }) {
       const modalities_list = modalities.split('\\');
 
       // Slide Microscopy modality not supported by basic mode yet
-      return !modalities_list.includes('SM');
+      return !!modalities_list.filter(modality => modality != 'SM' && modality !== 'ECG').length;
```

Comment:
This fixes some display issues when viewing a study that is viewable, but the mode failed to load because the test was incorrect.  Also remove ECG from a default view mode, as that just causes problems.

Reply:
We don't have any ECG in the viewer (public OHIF repo), and I'm pretty sure that will raise more questions that where is the ECG data and where is the viewer that can load them

Reply:
There doesn't have to be - the issue is that these are not viewable by the viewer right now, so it shouldn't attempt to load them.  However, I can revert this change, or else make a list of disallowed modalities.

Reply:
Changed to use NON_IMAGE_MODALITIES list and .indexOf, so it is clear what is going on.